src/components/Steps/StepsGalleryHeader.js

```diff
@@ -0,0 +1,37 @@
+import React, { useCallback, useMemo } from "react";
+import { StepsDot } from "./StepsDot";
+import { STEPS_CSS_BASE_CLASS } from "./StepsConstants";
+
+const CSS_BASE_CLASS = `${STEPS_CSS_BASE_CLASS}-header_gallery`;
+
+export const StepsGalleryHeader = ({
+  activeStepIndex,
+  stepsCount,
+  onChangeActiveStep,
+  groupAriaLabel,
+  stepDescriptionFunc
+}) => {
+  const defaultStepDescriptionFunc = useCallback(stepIndex => `Step number ${stepIndex}`, []);
+  const galleryDots = useMemo(() => {
+    const dots = [];
+    for (let stepIndex = 0; stepIndex < stepsCount; stepIndex++) {
+      dots.push(
+        <StepsDot
+          key={`monday-style-step-dot-${stepIndex + 1}`}
```

Comment:
this is not unique, please check if we can have uniqueness here?

Reply:
i understand it is a bad practice but every solution i find is a bit hard  for this case(I dont know  what is the value of each step content - if the the value is a div or a react component or a string)